CMakeLists.txt

```diff
@@ -37,8 +37,9 @@ ExternalProject_Add(CoreFoundation
                       -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}
                       -DCMAKE_C_COMPILER=${CMAKE_C_COMPILER}
                       # NOTE(compnerd) ensure that we build it as a DLL as we
-                      # wish to re-export the symbols from Foundation
-                      -DCMAKE_C_FLAGS=-D_USRDLL
```

Comment:
Passing `-DCMAKE_C_FLAGS=` here completely overwrites the `CMAKE_C_FLAGS` value for the entire CMake invocation used to build CoreFoundation. This is not what you want; in fact, it actually corrupts the existing logic already in `CoreFoundation/CMakeLists.txt` to do this. The change you want here is to delete these three lines entirely.